apps/website/src/components/shared/form/UploadAttachmentsField.tsx

```diff
@@ -186,8 +186,8 @@ export const UploadAttachmentsField = ({
       }
 
       newFiles.push(
-        new Promise(async (resolve) => {
-          const dataURL = await fileToBase64(file);
+        new Promise((resolve) => {
```

Comment:
**Do not apply directly**

We do need to await the `fileToBase64()` call. I think this should work:

```suggestion
        (async () => {
          const dataURL = await fileToBase64(file);
          return {
            id: `upload-${fileCounter++}`,
            status: "upload.pending",
            dataURL,
            file,
          };
        })(),
```